.env.development

```diff
@@ -7,4 +7,7 @@ REACT_APP_ACTIVITIES_ON=false
 REACT_APP_ACTIVITY_ID=2playsamonth
 REACT_APP_DADJOKES_URL=https://jokeapi-v2.p.rapidapi.com/joke/
 REACT_APP_DADJOKES_APIKEY='b71df95c75msha446fab91d0e935p1d0262jsn1d938cb85502'
-REACT_APP_DADJOKES_APIHOST='jokeapi-v2.p.rapidapi.com'
\ No newline at end of file
+REACT_APP_DADJOKES_APIHOST='jokeapi-v2.p.rapidapi.com'
```

Comment:
You need to provide these keys to us to add to the vercel.

Also you don't have to commit them. You can keep it in your .env.local